docs/prerequisites.adoc

```diff
@@ -80,7 +70,7 @@ Open a terminal and type the following:
 sudo mv /path/to/terraform /usr/local/bin
```

Comment:
Never move binaries. That's not how you configure the path. Either update `.bashrc` with something like `export PATH=/path/to/terraform/bin:$PATH` or create a symbolic link  (which is what Homebrew does).

Reply:
I think the `configure your path` sections should be simply removed. If the user decides to go with manual installation, they can follow the instruction from terraform page as you suggested and manage their operating system as they prefer.

Terraform install page simply says the following, with a link to StackExchange for information related to `PATH`.

>Terraform is distributed as a single binary. Install Terraform by unzipping it and moving it to a directory included in your system's PATH .